task-sdk/src/airflow/sdk/api/client.py

```diff
@@ -82,6 +82,7 @@ def wrapper(f):
             return f
 
         return wrapper
+
```

Comment:
small nit: do we need this additional space, looks like this is the only change to this file?

Reply:
We don't, let me rectify it

Reply:
done